beamer/models/request.py

```diff
@@ -37,8 +37,10 @@ def __init__(
         self.fill_tx: Optional[HexBytes] = None
         self.fill_timestamp: Optional[Timestamp] = None
         self.fill_id: Optional[FillId] = None
+        self.invalid_fill_ids: set[tuple[FillId, HexBytes, int]] = set()
```

Comment:
I'm thinking this might be better as a mapping `FillId` -> `tuple[HexBytes, int]`.